glow_test.go

```diff
@@ -16,13 +18,21 @@ func TestGlowSources(t *testing.T) {
 
 	for _, v := range tt {
 		t.Run(v, func(t *testing.T) {
+			var urlErr *net.DNSError
 			buf := &bytes.Buffer{}
-			err := executeArg(rootCmd, v, buf)
+			_, err := sourceFromArg(v)
 			if err != nil {
-				t.Errorf("Error during execution (args: %s): %v", v, err)
-			}
-			if buf.Len() == 0 {
-				t.Errorf("Output buffer should not be empty (args: %s)", v)
+				// Check for network issues.
+				if errors.As(err, &urlErr) {
+					t.Logf("Error during execution (args: %s):\n%v", v, err)
+					t.Skip("Test uses network. Are you connected to the Internet?")
+				} else {
+					t.Errorf("Error during execution (args: %s): %T %v", v, err, err)
+				}
+
+				if buf.Len() == 0 {
+					t.Errorf("Output buffer should not be empty (args: %s)", v)
+				}
```

Comment:
This could be easier to read

```suggestion
				// Check for network issues.
				if errors.As(err, &urlErr) {
					t.Logf("Error during execution (args: %s):\n%v", v, err)
					t.Skip("Test uses network. Are you connected to the Internet?")
					return
				}
				t.Errorf("Error during execution (args: %s): %T %v", v, err, err)

				if buf.Len() == 0 {
					t.Errorf("Output buffer should not be empty (args: %s)", v)
				}
```

Reply:
Ah, I'm just debugging github actions right now, will be undoing these commits once I figure out what's going on 😅